numpy/typing/_add_docstring.py

```diff
@@ -0,0 +1,96 @@
+"""A module for creating docstrings for sphinx ``data`` domains."""
+
+import re
+import textwrap
+
+_docstrings_list = []
+
+
+def add_newdoc(name, value, doc):
+    _docstrings_list.append((name, value, doc))
+
+
+def _parse_docstrings():
```

Comment:
Based on [`np.doc.constants`](https://github.com/numpy/numpy/blob/master/numpy/doc/constants.py), which uses a approach in generating documentation for various constants.